pkg/logs/input/container/docker.go

```diff
@@ -208,6 +227,34 @@ func (dt *DockerTailer) forwardMessages() {
 	}
 }
 
+// fetchLogsMetadata returns the logs metedata defined in the container labels.
+// if no metadata are specified, return false
+func (dt *DockerTailer) fetchLogsMetadata() (*config.LogsConfig, bool) {
+	information, err := dt.cli.ContainerInspect(context.Background(), dt.ContainerID)
+	if err != nil {
+		return nil, false
+	}
+	label, exists := information.Config.Labels[logsMetadataPath]
+	if !exists {
+		return nil, false
+	}
+	return dt.computeLogsMetadataFromLabel(label)
+}
+
+// computeLogsMetadataFromLabel returns logs metadata if label is correctly formatted.
+func (dt *DockerTailer) computeLogsMetadataFromLabel(label string) (*config.LogsConfig, bool) {
+	var configs []config.LogsConfig
```

Comment:
do we want to have a `[]config.LogConfig` or simply a `config.LogConfig`?

Reply:
we expect the label to be like:
```
LABEL "com.datadoghq.ad.logs"='[{"source": "any_source", "service": "any_service"}]'
```

Reply:
It's worthy to add a description though